log/loki.go

```diff
@@ -150,10 +150,13 @@ func (h *lokiHook) Listen(ctx context.Context) {
 		pushCh     = make(chan chan int64)
 	)
 
+	pushDone := make(chan struct{})
+	defer func() { <-pushDone }()
```

Comment:
I did use a sync.WaitGroup in thePoC, but decided there is no need to import the `sync` package and this code is even shorter so :shrug: